app/scienceapi/events/tests/test_views.py

```diff
@@ -0,0 +1,205 @@
+import json
+from django.core.urlresolvers import reverse
+from django.test import TestCase
+from urllib.parse import urlencode
+
+from scienceapi.events.serializers import (
+    EventSerializer,
+    EventProjectSerializer,
+    EventExpandAllSerializer,
+)
+from scienceapi.users.tests.test_models import UserFactory
+from scienceapi.events.tests.test_models import EventFactory
+from scienceapi.projects.tests.test_models import (
+    ProjectFactory,
+    UserProjectFactory,
+)
+
+
+class TestEventView(TestCase):
+
+    def create_user_project(self, user, project):
```

Comment:
Can we move all these helper functions to the `utility` folder in a new file called `test_models` so that we don't have repeated code?